WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailContainerFragment.java

```diff
@@ -66,27 +90,102 @@ public static HistoryDetailContainerFragment newInstance(Revision revision, Arra
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         ViewGroup rootView = (ViewGroup) inflater.inflate(R.layout.history_detail_container_fragment, container, false);
 
+        mIsFragmentRecreated = savedInstanceState != null;
+
         if (getArguments() != null) {
             mRevision = getArguments().getParcelable(EXTRA_REVISION);
             mRevisions = getArguments().getParcelableArrayList(EXTRA_REVISIONS);
         }
 
-        mIsFragmentRecreated = savedInstanceState != null;
+        mPosition = mRevisions.indexOf(mRevision);
```

Comment:
This is an interesting question :) I'm thinking about a situation where `mRevisions` might be `null`, and theoretically, it could happen when something really bad happened. I think there is a benefit in letting app crash (maybe throw exception ourself with the proper error message), so we can notice it earlier and address the issue. What do you think?